src/System.ServiceModel.Primitives/src/System.ServiceModel.Primitives.Facade.csproj

```diff
@@ -1,13 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyVersion>4.6.0.0</AssemblyVersion>
+    <!--<TargetFrameworks>MonoAndroid10;MonoTouch10;net45;net46;net461;netcore50;netcoreapp2.1;netstandard1.0;netstandard1.1;netstandard1.3;netstandard2.0;portable-net45+win8+wp8;win8;wp8;xamarinios10;xamarinmac20;xamarintvos10;xamarinwatchos10</TargetFrameworks>-->
     <TargetFrameworks>MonoAndroid10;MonoTouch10;net45;net46;net461;netcore50;netstandard1.0;netstandard1.1;netstandard1.3;netstandard2.0;portable-net45+win8+wp8;win8;wp8;xamarinios10;xamarinmac20;xamarintvos10;xamarinwatchos10</TargetFrameworks>
     <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
     <CLSCompliant>true</CLSCompliant>
     <IsPackable>true</IsPackable>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1' Or '$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
You may want to try something like:
```xml
<BuildFrameworks>netstandard;netcoreapp2.1;</BuildFrameworks>
<HarvestFrameworks>MonoAndroid10; ...;</HarvestFrameworks>
<TargetFrameworks>$(BuildFrameworks)$(HarvestFrameworks)</TargetFrameworks>
<IsBuilding Condition="$(BuildFrameworks.Contains('$(TargetFramework);'))">true</IsBuilding>
<IsHarvesting Condition="$(BuildFrameworks.Contains('$(TargetFramework);'))">true</IsHarvesting>
```

That could simplify conditions a little.  Up to you.

Reply:
I like that, it provides built-in clarity as well.